tssh/agent.go

```diff
@@ -80,46 +80,17 @@ func getAgentClient(args *sshArgs, param *sshParam) agent.ExtendedAgent {
 		agentClient = agent.NewClient(conn)
 		debug("new ssh agent client [%s] success", addr)
 
-		afterLoginFuncs = append(afterLoginFuncs, func() {
-			conn.Close()
-			agentClient = nil
-		})
+		if args.ForwardAgent {
```

Comment:
There are also `args.NoForwardAgent` and `strings.ToLower(getOptionConfig(args, "ForwardAgent")) == "yes"`.